src/newViews/PillarAnalysis/EntriesFilterForm/index.tsx

```diff
@@ -124,24 +133,24 @@ function EntriesFilterForm(props: OwnProps) {
         failureHeader: _ts('pillarAnalysis', 'entryOptions'),
     });
 
-    const schema = useMemo(() => ({
-        fields: {
-            created_at: [],
-            created_by: [],
-            comment_assignee: [],
-            comment_created_by: [],
-            comment_status: [],
-            verified: [],
-            entry_type: [],
-            project_entry_labels: [],
-            lead_group_label: [],
-            ...listToMap(filters, v => v.key, () => []),
-        },
-    }), [filters]);
+    const {
+        pristine,
+        validate,
+        onErrorSet,
+        value,
+        onValueSet,
+        onValueChange,
+    } = useForm(initialValue, schema);
+
+    useEffect(() => {
+        if (filtersValue) {
+            onValueSet(filtersValue);
+        }
```

Comment:
Why add this `if` check

Reply:
we shouldn't set the form value as undefined which might be the case

Reply:
why not?